fluid-music/src/vst2ToReaperObject.ts

```diff
@@ -0,0 +1,83 @@
+import { FluidPlugin, PluginType } from './plugin';
+import FluidIpcClient = require('./cybr/IpcClient');
+
+const rppp = require('rppp')
+const fluid = require('./cybr/index');
+
+/**
+ * Create a `ReaperVst` object from an existing plugin on the cybr instance.
+ *
+ * @param {FluidIpcClient} client A FluidIpcClient that is connected to a cybr instance
+ * @param {string} trackName - See documentation for fluid.audiotrack.select()
+ * @param {string} pluginName - See documentation for fluid.plugin.select()
+ * @param {string} pluginType
+ * @param {string} pluginId 
+ * 
```

Comment:
Looks like the JSDoc style documentation is out of date - the arguments are not strings any more. TypeScript uses a similar syntax to JSDoc, but it is not necessary to specify the types in the `@param` statement ( ex. `{string}`), because the type information is written explicit in the function signature.